FakeNFT/Presentation/ProfileFlow/NFTCollection/NFTCollectionCell.swift

```diff
@@ -0,0 +1,176 @@
+import UIKit
+
+final class NFTCollectionCell: UITableViewCell {
+
+    static let reuseIdentifier = "nftCell"
+
+    private enum Constants {
+        static let nftImageSize: CGFloat = 108
+        static let likeButtonSize: CGFloat = 42
+        static let vInset: CGFloat = 8                  // vertical content inset
+        static let hInset: CGFloat = 16                 // horizontal content inset
+    }
+
+    private lazy var nftImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.cornerRadius = 12
+        imageView.layer.masksToBounds = true
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.backgroundColor = .black
+        return imageView
+    }()
+
+    private lazy var likeButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: "NotLiked"), for: .normal)
```

Comment:
Можно воспользоваться точечной нотацией из файла UIImage+Extensions, чтобы получить значение ассета и не допустить случайной опечатки при вводе текстом, например
```suggestion
        button.setImage(UIImage.NFTIcon.notLiked, for: .normal)
```